lib/metadata/schema-parser.js

```diff
@@ -114,9 +124,11 @@ SchemaParser.prototype.getTable = function (keyspaceName, name, cache, callback)
   tableInfo.loading = true;
   let tableRow, columnRows, indexRows;
   const self = this;
+  let virtualTable = virtual;
   utils.series([
     function getTableRow(next) {
-      const query = util.format(self.selectTable, keyspaceName, name);
+      const selectTable = virtualTable ? _selectVirtualTable : self.selectTable; 
+      const query = util.format(selectTable, keyspaceName, name);
       self.cc.query(query, function (err, response) {
         if (err) {
           return next(err);
```

Comment:
Unlike other cases we error out here for virtual tables because in this case we know we are dealing with a table in a virtual keyspace, so propagating any errors seems sensible.